src/interpreter.rs

```diff
@@ -1144,9 +1165,10 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
         let vtable = selection.map(|predicate| {
             fulfill_cx.register_predicate_obligation(&infcx, predicate);
         });
-        let vtable = infer::drain_fulfillment_cx_or_panic(
-            DUMMY_SP, &infcx, &mut fulfill_cx, &vtable
-        );
+        let vtable = infer::drain_fulfillment_cx_or_panic(DUMMY_SP,
+                                                          &infcx,
+                                                          &mut fulfill_cx,
+                                                          &vtable);
```

Comment:
Again, I don't want to use alignment. Rust is already bad when it comes to rightwards drift and this exacerbates it immensely whenever there's a non-trivial expression in an argument list.

See https://github.com/rust-lang-nursery/rustfmt/issues/439.